app/templates/components/conversations/conversation-part-closed.hbs

```diff
@@ -0,0 +1,9 @@
+<div class="conversation-part-closed">
+  <div class="coversation-part-closed__message-wrapper">
+    {{if currentUser}}
+    You closed this at...
+    {{else}}
+    <div class="conversation-part-closed__message-wrapper">
+      {{author.username}} closed this at...
```

Comment:
See comment above on what to do here. Also, your syntax is incorrect. You use the `if` helper like this:

```
{{#if something}}
  Foo
{{else}}
  Bar
{{/if}}

Note the indentations, the `#` and the `{{/if}}`